packages/bezier-react/src/components/AlphaStatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { type CSSProperties, forwardRef, memo } from 'react'
+
+import { MoonFilledIcon } from '@channel.io/bezier-icons'
+import classNames from 'classnames'
+
+import { cssVar } from '~/src/utils/style'
+
+import { Icon } from '~/src/components/Icon'
+
+import { type StatusBadgeProps } from './StatusBadge.types'
+
+import styles from './StatusBadge.module.scss'
+
+/**
+ * `StatusBadge` is a component to indicate user status.
+ * @example
+ *
+ * ```tsx
+ * <StatusBadge
+ *   size="m"
+ *   online
+ *   doNotDisturb
+ * />
+ * ```
+ */
+export const StatusBadge = memo(
+  forwardRef<HTMLDivElement, StatusBadgeProps>(function StatusBadge(
+    {
+      size = 'm',
+      online = false,
+      doNotDisturb = false,
+      style,
+      className,
+      ...rest
+    },
+    forwardedRef
+  ) {
+    const withIcon = doNotDisturb
+    const iconColor = online
```

Comment:
(마이너) 타입이 있으면 좋을 것 같아요 (L44도)

```suggestion
import { type SemanticColor } from '~/src/types/tokens'

const iconColor: SemanticColor = online
```